compose/aspnet-mssql-compose.md

```diff
@@ -0,0 +1,142 @@
+
+1. Create a `docker-compose.yml` file. Write the following in the file, and make sure to replace the password in the `SA_PASSWORD` environment variable under `db` below. This file will define the way the images will interact as micro-services. 
+
+    >**Note**: SQL Server requires a secure password (Minimum length 8 characters, including uppercase and lowercase letters, base 10 digits and/or non-alphanumeric symbols).
+
+    ```
+    version: '3'
+
+    services:
+        web:
+            build: .
+            ports: 
+                - "8000:80"
+            depends_on:
+                - db
+        db:
+            image: "microsoft/mssql-server-linux"
+            environment:
+                SA_PASSWORD: "your_password"
+                ACCEPT_EULA: "Y"
+    ```
+
+    This file defines the `web` and `db` micro-services, their relationship, the ports they are using, and their specific environment variables.
+
+1. Go to `Startup.cs` and locate the function called `ConfigureServices` (Hint: it should be under line 42). Replace the entire function to use the following code (watch out for the brackets!).
+
+    >**Note**: Make sure to update the `Password` field in the `connection` variable below to the one you defined in the `docker-compose.yml` file.
+
+    ```csharp
+    [...]
+    public void ConfigureServices(IServiceCollection services)
+    {
+        // Database connection string. 
+        // Make sure to update the Password value below from "your_password" to your actual password.
+        var connection = @"Server=db;Database=master;User=sa;Password=your_password;";
+        
+        // This line uses 'UseSqlServer' in the 'options' parameter
+        // with the connection string defined above.
+        services.AddDbContext<ApplicationDbContext>(
+            options => options.UseSqlServer(connection));
+
+        services.AddIdentity<ApplicationUser, IdentityRole>()
+            .AddEntityFrameworkStores<ApplicationDbContext>()
+            .AddDefaultTokenProviders();
+
+        services.AddMvc();
+
+        // Add application services.
+        services.AddTransient<IEmailSender, AuthMessageSender>();
+        services.AddTransient<ISmsSender, AuthMessageSender>();
+    }
+    [...]
+    ```
+
+1. Ready! You can now run the `docker-compose build` command.
+
+    ```bash
+    docker-compose build
+    ```
+
+1. Run the `docker-compose up` command. After a few seconds, you should be able to open [localhost:8000](http://localhost:8000) and see the ASP.NET core sample website. The application is listening on port 80 by default, but we mapped it to port 8000 in the `Dockerfile`.
+
+    ```bash
+    docker-compose up
+    ```
+
+    Go ahead and try out the website! This sample will use the SQL Server database image in the back-end for authentication.
+
+Ready! You now have a ASP.NET Core application running against SQL Server in Docker Compose!
+
+## Further reading
```

Comment:
Could we add something to highlight that Docker isn't just for .NET Core and Linux? Maybe link into the [Docker labs for Windows containers](https://github.com/docker/labs/tree/master/windows) which have full .NET Framework and SQL Server labs.